es/scripts/release.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+set -e
+
+VERSION=$(git describe --tags --abbrev=0 | cut -c 2-)
```

Comment:
I guess it would be better to rely on the actual tag that
this workflow is run with. On GitHub Actions, we can get the
tag name with `GITHUB_REF` environment variable.
https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables